internal/cmd/operator-sdk/cli/cli.go

```diff
@@ -89,6 +92,24 @@ func GetPluginsCLIAndRoot() (cli.CLI, *cobra.Command) {
 	}
 	root.PersistentPreRun = rootPersistentPreRun
 
+	// Hide `alpha config-gen` subcommand
+	// TODO: stop hiding `alpha config-gen` subcommand when it is more mature
+	var alpha *cobra.Command
+	for _, subcmd := range root.Commands() {
+		if subcmd.Name() == "alpha" {
+			alpha = subcmd
+			break
+		}
+	}
+	if alpha != nil {
+		for _, subcmd := range alpha.Commands() {
+			if subcmd.Name() == "config-gen" {
+				subcmd.Hidden = true
+				break
+			}
+		}
+	}
```

Comment:
In KB we merged the config-gen an alpha, however,  It is in a very very very init stage, it is not compatible with the language plugins and it is not covered with tests/ci, has not all. features or docs. We just merge to make easier allow it to be improved. So, for SDK, IMO we must hide that.

Reply:
That's why it's in alpha. I'd like to have it added to `operator-sdk`.

Reply:
Yes. But we have alpha commands as well. I think we need to wait for a little before allowing it indeed for alpha in the SDK side. So, IMO we ough to move forward with it hidden and then, we have the options to choose when we will provide it. See that the alpha commands also are described in the docs CLI/ section as well. 

we cannot ensure anything about it so far. However, if you folks think that is fine provide that. Then, I am. OK with.

Reply:
Hiding an alpha tool is counter-intuitive IMO. Don't you _want_ people to try out these alpha commands and give feedback? Anything that's marked 'alpha' is pretty clearly not stable.

Reply:
Want to mention that I hided it under request, I have no strong opinion here. Hiding means no docs and no exposure.

Reply:
Hi @estroz,

>  Don't you want people to try out these alpha commands and give feedback?

Regards the config-gen I am not sure if it is in a stage that would be nice to offer to the SDK users and get feedback. It is exactly the point/motivation for I asked for @adirio to hide that. 

However, I am ok with the final decision too.

Reply:
Will let someone else be the tiebreaker here. 

/ping @jmrodri @rashmigottipati

Reply:
PS.: We can make it available any time that we wish, just by removing the //todo: code that hides it as well.

Reply:
I'd say let's merge this PR as is, and we can decide why and how we want alpha config-gen later.

Reply:
#4670